flow/core/params.py

```diff
@@ -8,6 +8,14 @@
 from flow.controllers.rlcontroller import RLController
 from flow.controllers.lane_change_controllers import SimLaneChangeController
 
+# To add these after creating the forlder for energy models # (1)
+"""
+from flow.enermod.toyota_energy import PriusEnergy
+from flow.enermod.toyota_energy import TacomaEnergy
+from flow.enermod.power_demand import PDMCombustionEngine
+from flow.enermod.power_demand import PDMElectric
+"""
+
```

Comment:
Plan is to create a folder consisting of all the energy models. Temporarily i name it as "enermod".  This section is commented for now until the idea is approved and the folder has been created.

Reply:
yes this is fine. let's name the folder `energy_models`